main.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jul 23 22:33:29 2019
+
+@author: j-bd
+"""
+import os
+import logging
+
+import pandas as pd
+
+import pneumonia_functions
+import pneumonia_detection
+import constants
+
+
+logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.DEBUG)
+
+def pre_trainning(args):
+    '''Lauch all necessary steps to set up Yolo v3 algorithm before for objects
+    trainning'''
+    image_dir = args.origin_folder
+    input_train_data_dir = os.path.join(image_dir, "stage_2_train_images")
```

Comment:
Maybe you should turn `args` to a `pd.Series` and add all those paths to `args` in the `create_parser` function

Reply:
@Ngoguey42 I turned `args` into a `dict()` and then add all those paths. It looks far better. Thanks for the tips.